plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/Statistics.java

```diff
@@ -293,6 +293,8 @@ private static void setColumnStatisticsForEmptyPartition(Type columnType, HiveCo
             case MAX_VALUE:
                 setMinMaxForEmptyPartition(columnType, result);
                 return;
+            case NUMBER_OF_DISTINCT_VALUES_SUMMARY:
+                throw new TrinoException(HIVE_UNKNOWN_COLUMN_STATISTIC_TYPE, "Unsupported column statistics type: " + columnStatisticType.name());
```

Comment:
you can `break` here 
```
break; // unsupported
```

also maybe put this case after `NUMBER_OF_DISTINCT_VALUES`?